packages/transport/package.json

```diff
@@ -15,18 +15,19 @@
         "Trezor",
         "transport"
     ],
-    "main": "./lib/index.js",
+    "main": "./src/index.ts",
     "browser": {
-        "./lib/transports/nodeusb": "./lib/transports/nodeusb.browser",
-        "./lib/transports/udp": "./lib/transports/udp.browser",
-        "./lib/transports/webusb": "./lib/transports/webusb.browser",
         "./src/transports/nodeusb": "./src/transports/nodeusb.browser",
         "./src/transports/udp": "./src/transports/udp.browser",
         "./src/transports/webusb": "./src/transports/webusb.browser"
     },
-    "react-native": {
```

Comment:
I just wanted to make sure that the correct file `applyContentTypeHeader.native.js` will be applied for the native version. If are you confident, that this directive is not necessary and the file will be applied just because there is a `.native.js` suffix, feel free to remove it from here.